__tests__/app.test.js

```diff
@@ -68,3 +68,38 @@ describe("GET: /api/articles", () => {
       });
   });
 });
+
+describe("GET: /api/articles/:article_id", () => {
+  test("200: should return an array of articles of given id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
```

Comment:
We can be much more specific with the object that we expect back - you've made a request for article_id 1 - therefore you can test to see if you get that specific article. I.e. test against the article full string values from the data, not just to see if they match any string.

Reply:
Thanks Simon.Will update my code.